src/types/tmdb.ts

```diff
@@ -0,0 +1,56 @@
+export interface MovieData {
+    page: number;
+    results: MovieResultData[];
+    total_pages: number;
+    total_results: number;
+}
+
+export interface MovieResultData {
+    adult: boolean;
+    backdrop_path: string;
+    genre_ids: number[];
+    id: number;
+    original_language: string;
+    original_title: string;
+    overview: string;
+    popularity: number;
+    poster_pat: string;
+    release_date: string;
+    title: string;
+    video: boolean;
+    vote_average: number;
+    vote_count: number;
+}
+
+export interface MovieDetailsData {
+    adult: boolean,
+    backdrop_path: string,
+    belongs_to_collection: any[],
+    budget: number,
+    genres: any[]
```

Comment:
```suggestion
    genres: {
        id: number;
        name: string
    }
```

Reply:
We might be able to do with for production companies as well. Just nested interface rather than a new one.